project-management/src/main/java/life/qbic/projectmanagement/domain/model/project/purchase/Offer.java

```diff
@@ -0,0 +1,72 @@
+package life.qbic.projectmanagement.domain.model.project.purchase;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Lob;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Random;
+
+/**
+ * <b>Offer</b>
+ *
+ * An offer in the context of project management, not finance and accounting.
+ *
+ * @since 1.0.0
+ */
+@Entity(name = "purchase_offer")
+public class Offer {
```

Comment:
Nice. From the equals method I gather that this is a value object.

Reply:
exactly